.vscode/settings.json

```diff
@@ -5,6 +5,10 @@
     "editor.defaultFormatter": "mtxr.sqltools",
     "editor.formatOnSave": true
   },
+  "go.useLanguageServer": true,
+  "gopls": {
+    "formatting.gofumpt": true
```

Comment:
happy to see gofumpt here!

Reply:
CI depends on golangci. I usually format the code with `golangci-lint fmt`.

Here is the current config:
```
formatters:
  enable:
    - gofmt
    - golines
    - gofumpt
```

I would use the linter rather than the various formatters

Reply:
OK. I'll see if that has any other snags. I've updated the config to use `golangci-lint fmt` and things seem to be formatting.